paimon-core/src/test/java/org/apache/paimon/rest/auth/AuthSessionTest.java

```diff
@@ -101,8 +101,10 @@ public void testRefreshDLFAuthTokenFileAuthProvider() throws IOException, Interr
         tokenFile2Token = generateTokenAndWriteToFile(fileName);
         token = tokenFile2Token.getRight();
         Thread.sleep(tokenRefreshInMills * 2);
-        authToken = OBJECT_MAPPER_INSTANCE.writeValueAsString(dlfAuthProvider.token);
-        assertEquals(authToken, token);
+        if (dlfAuthProvider.refreshSuccess) {
```

Comment:
Maybe you can get token first, and get token to see if it is be equal to previous token.